engine/access/rest/accounts.go

```diff
@@ -29,5 +46,46 @@ func getAccount(r *requestDecorator, backend access.API, link LinkGenerator) (in
 		}
 	}
 
-	return accountResponse(account), nil
+	expandContracts := r.expands(expandableFieldContracts)
+	expandKeys := r.expands(expandableFieldKeys)
+	return accountResponse(account, link, expandKeys, expandContracts)
+}
+
+func accountResponse(flowAccount *flow.Account, link LinkGenerator, expandKeys bool, expandContracts bool) (generated.Account, error) {
```

Comment:
@sideninja I moved account response here from converter. I didn't find a good way of not passing the link generator tbh. Passing it actually makes testing easy since its injected.
Also, implemented the `if-then-else` logic for the expandables - `keys` and `contracts`

Reply:
How come that has been moved here? I feel that it's getting more confusing if we do some response creation here and some in converters. I always prefer to have one place to do a certain functionality so we either have handlers to do also result generation or we keep converters to do that.

Reply:
If that was done due to expanded keys then we can just call `accountResponse` and remove result values if expanded is not passed like so:
```go
account := accountResponse(account, link)

if !r.expands(keys) {
    account.keys = nil
}
```
But I would rather have all of that in converters as that is one place that is generating the response, but I think that passing objects to it is not the best idea so I agree we should change it to support booleans like `expandKeys`, we can even pass the link if you think that is better and generate it in the handler.

Reply:
I like the idea of having the accountResponse in the converter and passing an `expand` map to let it decide what fields to expand. I have modified the code accordingly.